static/docs/command-reference/remote/modify.md

```diff
@@ -27,8 +27,8 @@ positional arguments:
 ## Description
 
 Remote `name` and `option` name are required. Option names are remote type
-specific. See below examples and a list of remote storage types: Amazon S3,
-Google Cloud, Azure, Google Drive, SSH, ALiyun OSS, among others.
+specific. See `dvc remote add` and **Available settings** section below for a
```

Comment:
Available setting - make a link? (and remove emphasize?)

Reply:
Done in 884968d9.